### ISSUE TITLE
GLPI 10.0.0-rc3 Strange characters in tickets statistics

### ISSUE THREAD

**Issue by bolivarf** [glpi-project/glpi#11123]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

GLPI 10.0.0-rc3

### Bug description

good morning
the ticket statistics appear some strange characters 

in Firefox
![Captura de pantalla de 2022-03-28 06-27-51](https://user-images.githubusercontent.com/7965864/160387424-a69d98f6-0c62-4377-bdbf-57473ea2299b.png)

in Google Chrome
![Captura de pantalla de 2022-03-28 06-27-11](https://user-images.githubusercontent.com/7965864/160387429-67ed168c-5eb8-4fa8-a1d8-008fd7c509b1.png)

I thought it was something from css try to execute
bin/console glpi:build:compile_scss
but it gives me the following error:

PHP Warning (2): file(tools/HEADER): failed to open stream: No such file or directory in src/Console/Build/CompileScssCommand.php at line 171
PHP Warning:  file(tools/HEADER): failed to open stream: No such file or directory in src/Console/Build/CompileScssCommand.php on line 171
PHP Warning (2): array_map(): Expected parameter 2 to be an array, bool given in src/Console/Build/CompileScssCommand.php at line 175
PHP Warning:  array_map(): Expected parameter 2 to be an array, bool given in src/Console/Build/CompileScssCommand.php on line 175
PHP Warning (2): array_merge(): Expected parameter 2 to be an array, null given in src/Console/Build/CompileScssCommand.php at line 183
PHP Warning:  array_merge(): Expected parameter 2 to be an array, null given in src/Console/Build/CompileScssCommand.php on line 183
PHP Warning (2): implode(): Argument must be an array in src/Console/Build/CompileScssCommand.php at line 185
PHP Warning:  implode(): Argument must be an array in src/Console/Build/CompileScssCommand.php on line 185
Uncaught Exception TypeError: Return value of Glpi\Console\Build\CompileScssCommand::getLicenceHeaderString() must be of the type string, null returned in src/Console/Build/CompileScssCommand.php at line 185

### Relevant log output

```shell
...
[2022-03-28 06:34:07] glpiphplog.WARNING:   *** PHP Warning (2): array_merge(): Expected parameter 2 to be an array, null given in src/Console/Build/CompileScssCommand.php at line 183
  Backtrace :
  src/Console/Build/CompileScssCommand.php:183       array_merge()
  src/Console/Build/CompileScssCommand.php:120       Glpi\Console\Build\CompileScssCommand->getLicenceHeaderString()
  vendor/symfony/console/Command/Command.php:298     Glpi\Console\Build\CompileScssCommand->execute()
  vendor/symfony/console/Application.php:1015        Symfony\Component\Console\Command\Command->run()
  src/Console/Application.php:254                    Symfony\Component\Console\Application->doRunCommand()
  vendor/symfony/console/Application.php:299         Glpi\Console\Application->doRunCommand()
  vendor/symfony/console/Application.php:171         Symfony\Component\Console\Application->doRun()
  bin/console:119                                    Symfony\Component\Console\Application->run()
  
[2022-03-28 06:34:07] glpiphplog.WARNING:   *** PHP Warning (2): implode(): Argument must be an array in src/Console/Build/CompileScssCommand.php at line 185
  Backtrace :
  src/Console/Build/CompileScssCommand.php:185       implode()
  src/Console/Build/CompileScssCommand.php:120       Glpi\Console\Build\CompileScssCommand->getLicenceHeaderString()
  vendor/symfony/console/Command/Command.php:298     Glpi\Console\Build\CompileScssCommand->execute()
  vendor/symfony/console/Application.php:1015        Symfony\Component\Console\Command\Command->run()
  src/Console/Application.php:254                    Symfony\Component\Console\Application->doRunCommand()
  vendor/symfony/console/Application.php:299         Glpi\Console\Application->doRunCommand()
  vendor/symfony/console/Application.php:171         Symfony\Component\Console\Application->doRun()
  bin/console:119                                    Symfony\Component\Console\Application->run()
  
[2022-03-28 06:34:07] glpiphplog.CRITICAL:   *** Uncaught Exception TypeError: Return value of Glpi\Console\Build\CompileScssCommand::getLicenceHeaderString() must be of the type string, null returned in src/Console/Build/CompileScssCommand.php at line 185
  Backtrace :
  src/Console/Build/CompileScssCommand.php:120       Glpi\Console\Build\CompileScssCommand->getLicenceHeaderString()
  vendor/symfony/console/Command/Command.php:298     Glpi\Console\Build\CompileScssCommand->execute()
  vendor/symfony/console/Application.php:1015        Symfony\Component\Console\Command\Command->run()
  src/Console/Application.php:254                    Symfony\Component\Console\Application->doRunCommand()
  vendor/symfony/console/Application.php:299         Glpi\Console\Application->doRunCommand()
  vendor/symfony/console/Application.php:171         Symfony\Component\Console\Application->doRun()
  bin/console:119                                    Symfony\Component\Console\Application->run()
  
[2022-03-28 06:43:14] glpiphplog.WARNING:   *** PHP Warning (2): file(/home/fran/public_html/glpi10/tools/HEADER): failed to open stream: No such file or directory in /home/fran/public_html/glpi10/src/Console/Build/CompileScssCommand.php at line 171
  Backtrace :
  src/Console/Build/CompileScssCommand.php:171       file()
  src/Console/Build/CompileScssCommand.php:120       Glpi\Console\Build\CompileScssCommand->getLicenceHeaderString()
  vendor/symfony/console/Command/Command.php:298     Glpi\Console\Build\CompileScssCommand->execute()
  vendor/symfony/console/Application.php:1015        Symfony\Component\Console\Command\Command->run()
  src/Console/Application.php:254                    Symfony\Component\Console\Application->doRunCommand()
  vendor/symfony/console/Application.php:299         Glpi\Console\Application->doRunCommand()
  vendor/symfony/console/Application.php:171         Symfony\Component\Console\Application->doRun()
  bin/console:119                                    Symfony\Component\Console\Application->run()
  
[2022-03-28 06:43:14] glpiphplog.WARNING:   *** PHP Warning (2): array_map(): Expected parameter 2 to be an array, bool given in /home/fran/public_html/glpi10/src/Console/Build/CompileScssCommand.php at line 175
  Backtrace :
  src/Console/Build/CompileScssCommand.php:175       array_map()
  src/Console/Build/CompileScssCommand.php:120       Glpi\Console\Build\CompileScssCommand->getLicenceHeaderString()
  vendor/symfony/console/Command/Command.php:298     Glpi\Console\Build\CompileScssCommand->execute()
  vendor/symfony/console/Application.php:1015        Symfony\Component\Console\Command\Command->run()
  src/Console/Application.php:254                    Symfony\Component\Console\Application->doRunCommand()
  vendor/symfony/console/Application.php:299         Glpi\Console\Application->doRunCommand()
  vendor/symfony/console/Application.php:171         Symfony\Component\Console\Application->doRun()
  bin/console:119                                    Symfony\Component\Console\Application->run()
  
[2022-03-28 06:43:14] glpiphplog.WARNING:   *** PHP Warning (2): array_merge(): Expected parameter 2 to be an array, null given in /home/fran/public_html/glpi10/src/Console/Build/CompileScssCommand.php at line 183
  Backtrace :
  src/Console/Build/CompileScssCommand.php:183       array_merge()
  src/Console/Build/CompileScssCommand.php:120       Glpi\Console\Build\CompileScssCommand->getLicenceHeaderString()
  vendor/symfony/console/Command/Command.php:298     Glpi\Console\Build\CompileScssCommand->execute()
  vendor/symfony/console/Application.php:1015        Symfony\Component\Console\Command\Command->run()
  src/Console/Application.php:254                    Symfony\Component\Console\Application->doRunCommand()
  vendor/symfony/console/Application.php:299         Glpi\Console\Application->doRunCommand()
  vendor/symfony/console/Application.php:171         Symfony\Component\Console\Application->doRun()
  bin/console:119                                    Symfony\Component\Console\Application->run()
  
[2022-03-28 06:43:14] glpiphplog.WARNING:   *** PHP Warning (2): implode(): Argument must be an array in /home/fran/public_html/glpi10/src/Console/Build/CompileScssCommand.php at line 185
  Backtrace :
  src/Console/Build/CompileScssCommand.php:185       implode()
  src/Console/Build/CompileScssCommand.php:120       Glpi\Console\Build\CompileScssCommand->getLicenceHeaderString()
  vendor/symfony/console/Command/Command.php:298     Glpi\Console\Build\CompileScssCommand->execute()
  vendor/symfony/console/Application.php:1015        Symfony\Component\Console\Command\Command->run()
  src/Console/Application.php:254                    Symfony\Component\Console\Application->doRunCommand()
  vendor/symfony/console/Application.php:299         Glpi\Console\Application->doRunCommand()
  vendor/symfony/console/Application.php:171         Symfony\Component\Console\Application->doRun()
  bin/console:119                                    Symfony\Component\Console\Application->run()
  
[2022-03-28 06:43:14] glpiphplog.CRITICAL:   *** Uncaught Exception TypeError: Return value of Glpi\Console\Build\CompileScssCommand::getLicenceHeaderString() must be of the type string, null returned in /home/fran/public_html/glpi10/src/Console/Build/CompileScssCommand.php at line 185
  Backtrace :
  src/Console/Build/CompileScssCommand.php:120       Glpi\Console\Build\CompileScssCommand->getLicenceHeaderString()
  vendor/symfony/console/Command/Command.php:298     Glpi\Console\Build\CompileScssCommand->execute()
  vendor/symfony/console/Application.php:1015        Symfony\Component\Console\Command\Command->run()
  src/Console/Application.php:254                    Symfony\Component\Console\Application->doRunCommand()
  vendor/symfony/console/Application.php:299         Glpi\Console\Application->doRunCommand()
  vendor/symfony/console/Application.php:171         Symfony\Component\Console\Application->doRun()
  bin/console:119                                    Symfony\Component\Console\Application->run()
  
[2022-03-28 07:11:32] glpiphplog.WARNING: Test logger
```

### Steps To reproduce

1. Login to GLPI
2. Select Support > Tickets
3. Choose a ticket preferably with expired times
4. select the Statistics tab

### Your GLPI setup information

<details><summary>Information about system installation and configuration</summary><pre>GLPI 10.0.0-rc3 (/~test/glpi10 => /home/test/public_html/glpi10)
Installation mode: TARBALL
Current language:en_GB

</details><details><summary>Server</summary><pre> 
Operating system: Linux eqper01 5.13.0-37-generic #42~20.04.1-Ubuntu SMP Tue Mar 15 15:44:28 UTC 2022 x86_64
PHP 7.4.3 apache2handler (Core, FFI, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, apache2handler, apc, apcu, bz2,
	calendar, ctype, curl, date, dom, exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, intl, json, ldap, libxml, mbstring,
	mysqli, mysqlnd, openssl, pcre, pdo_mysql, pdo_pgsql, pgsql, posix, readline, session, shmop, sockets, sodium, standard,
	sysvmsg, sysvsem, sysvshm, tokenizer, xml, xmlreader, xmlrpc, xmlwriter, xsl, zip, zlib)
Setup: max_execution_time="30" memory_limit="128M" post_max_size="8M" safe_mode="" session.save_handler="files"
	upload_max_filesize="2M" 
Software: Apache/2.4.41 (Ubuntu) (Apache/2.4.41 (Ubuntu) Server at localhost Port 80
)
	Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.82 Safari/537.36
Server Software: (Ubuntu)
	Server Version: 8.0.28-0ubuntu0.20.04.3
	Server SQL Mode: STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION
	Parameters: glpiadmin@localhost/glpi2
	Host info: Localhost via UNIX socket
	
PHP version (7.4.3) is supported.
Sessions configuration is OK.
Allocated memory is sufficient.
mysqli extension is installed.
Following extensions are installed: dom, fileinfo, json, simplexml.
curl extension is installed.
gd extension is installed.
intl extension is installed.
libxml extension is installed.
zlib extension is installed.
The constant SODIUM_CRYPTO_AEAD_XCHACHA20POLY1305_IETF_NPUBBYTES is present.
Database engine version (8.0.28) is supported.
The log file has been created successfully.
Write access to /home/test/glpi2/files/_cache has been validated.
Write access to /home/test/glpi2/config has been validated.
Write access to /home/test/glpi2/files/_cron has been validated.
Write access to /home/test/glpi2/files has been validated.
Write access to /home/test/glpi2/files/_dumps has been validated.
Write access to /home/test/glpi2/files/_graphs has been validated.
Write access to /home/test/glpi2/files/_lock has been validated.
Write access to /home/test/glpi2/files/_pictures has been validated.
Write access to /home/test/glpi2/files/_plugins has been validated.
Write access to /home/test/glpi2/files/_rss has been validated.
Write access to /home/test/glpi2/files/_sessions has been validated.
Write access to /home/test/glpi2/files/_tmp has been validated.
Write access to /home/test/glpi2/files/_uploads has been validated.
exif extension is installed.
ldap extension is installed.
openssl extension is installed.
zip extension is installed.
bz2 extension is installed.
Zend OPcache extension is installed.
Following extensions are installed: ctype, iconv, mbstring, sodium.
Write access to /home/test/glpi2/makeplace has been validated.
Timezones seems loaded in database.

</details><details><summary>GLPI constants</summary><pre> 
GLPI_ROOT: /home/test/public_html/glpi10
GLPI_CONFIG_DIR: /home/test/glpi2/config
GLPI_VAR_DIR: /home/test/glpi2/files
GLPI_LOG_DIR: /home/test/glpi2/log
GLPI_MARKETPLACE_DIR: /home/test/glpi2/makeplace
GLPI_USE_CSRF_CHECK: 1
GLPI_CSRF_EXPIRES: 7200
GLPI_CSRF_MAX_TOKENS: 100
GLPI_USE_IDOR_CHECK: 1
GLPI_IDOR_EXPIRES: 7200
GLPI_ALLOW_IFRAME_IN_RICH_TEXT: 
GLPI_TELEMETRY_URI: https://telemetry.glpi-project.org
GLPI_INSTALL_MODE: TARBALL
GLPI_NETWORK_MAIL: glpi@teclib.com
GLPI_NETWORK_SERVICES: https://services.glpi-network.com
GLPI_MARKETPLACE_ALLOW_OVERRIDE: 1
GLPI_MARKETPLACE_MANUAL_DOWNLOADS: 1
GLPI_USER_AGENT_EXTRA_COMMENTS: 
GLPI_DISABLE_ONLY_FULL_GROUP_BY_SQL_MODE: 1
GLPI_AJAX_DASHBOARD: 1
GLPI_CALDAV_IMPORT_STATE: 0
GLPI_DEMO_MODE: 0
GLPI_CENTRAL_WARNINGS: 1
GLPI_DOC_DIR: /home/test/glpi2/files
GLPI_CACHE_DIR: /home/test/glpi2/files/_cache
GLPI_CRON_DIR: /home/test/glpi2/files/_cron
GLPI_DUMP_DIR: /home/test/glpi2/files/_dumps
GLPI_GRAPH_DIR: /home/test/glpi2/files/_graphs
GLPI_LOCAL_I18N_DIR: /home/test/glpi2/files/_locales
GLPI_LOCK_DIR: /home/test/glpi2/files/_lock
GLPI_PICTURE_DIR: /home/test/glpi2/files/_pictures
GLPI_PLUGIN_DOC_DIR: /home/test/glpi2/files/_plugins
GLPI_RSS_DIR: /home/test/glpi2/files/_rss
GLPI_SESSION_DIR: /home/test/glpi2/files/_sessions
GLPI_TMP_DIR: /home/test/glpi2/files/_tmp
GLPI_UPLOAD_DIR: /home/test/glpi2/files/_uploads
GLPI_INVENTORY_DIR: /home/test/glpi2/files/_inventories
GLPI_NETWORK_REGISTRATION_API_URL: https://services.glpi-network.com/api/registration/
GLPI_MARKETPLACE_PLUGINS_API_URI: https://services.glpi-network.com/api/glpi-plugins/
GLPI_I18N_DIR: /home/test/public_html/glpi10/locales
GLPI_VERSION: 10.0.0-rc3
GLPI_SCHEMA_VERSION: 10.0.0-rc3@95da1cf53007a5c27622e99b82f61e98c6e97906
GLPI_MARKETPLACE_PRERELEASES: 1
GLPI_MIN_PHP: 7.4.0
GLPI_MAX_PHP: 8.2.0
GLPI_YEAR: 2022

</details><details><summary>Libraries</summary><pre> 
htmlawed/htmlawed version 1.2.6 in (/home/test/public_html/glpi10/vendor/htmlawed/htmlawed)
phpmailer/phpmailer version 6.6.0 in (/home/test/public_html/glpi10/vendor/phpmailer/phpmailer/src)
simplepie/simplepie version 1.5.8 in (/home/test/public_html/glpi10/vendor/simplepie/simplepie/library)
mpdf/mpdf in (/home/test/public_html/glpi10/vendor/mpdf/mpdf/src)
michelf/php-markdown in (/home/test/public_html/glpi10/vendor/michelf/php-markdown/Michelf)
true/punycode in (/home/test/public_html/glpi10/vendor/true/punycode/src)
iamcal/lib_autolink in (/home/test/public_html/glpi10/vendor/iamcal/lib_autolink)
sabre/dav in (/home/test/public_html/glpi10/vendor/sabre/dav/lib/DAV)
sabre/http in (/home/test/public_html/glpi10/vendor/sabre/http/lib)
sabre/uri in (/home/test/public_html/glpi10/vendor/sabre/uri/lib)
sabre/vobject in (/home/test/public_html/glpi10/vendor/sabre/vobject/lib)
laminas/laminas-i18n in (/home/test/public_html/glpi10/vendor/laminas/laminas-i18n/src)
laminas/laminas-servicemanager in (/home/test/public_html/glpi10/vendor/laminas/laminas-servicemanager/src)
monolog/monolog in (/home/test/public_html/glpi10/vendor/monolog/monolog/src/Monolog)
sebastian/diff in (/home/test/public_html/glpi10/vendor/sebastian/diff/src)
donatj/phpuseragentparser in (/home/test/public_html/glpi10/vendor/donatj/phpuseragentparser/src/UserAgent)
elvanto/litemoji in (/home/test/public_html/glpi10/vendor/elvanto/litemoji/src)
symfony/console in (/home/test/public_html/glpi10/vendor/symfony/console)
scssphp/scssphp in (/home/test/public_html/glpi10/vendor/scssphp/scssphp/src)
laminas/laminas-mail in (/home/test/public_html/glpi10/vendor/laminas/laminas-mail/src/Protocol)
laminas/laminas-mime in (/home/test/public_html/glpi10/vendor/laminas/laminas-mime/src)
rlanvin/php-rrule in (/home/test/public_html/glpi10/vendor/rlanvin/php-rrule/src)
blueimp/jquery-file-upload in (/home/test/public_html/glpi10/vendor/blueimp/jquery-file-upload/server/php)
ramsey/uuid in (/home/test/public_html/glpi10/vendor/ramsey/uuid/src)
psr/log in (/home/test/public_html/glpi10/vendor/psr/log/Psr/Log)
psr/simple-cache in (/home/test/public_html/glpi10/vendor/psr/simple-cache/src)
psr/cache in (/home/test/public_html/glpi10/vendor/psr/cache/src)
league/csv in (/home/test/public_html/glpi10/vendor/league/csv/src)
mexitek/phpcolors in (/home/test/public_html/glpi10/vendor/mexitek/phpcolors/src/Mexitek/PHPColors)
guzzlehttp/guzzle in (/home/test/public_html/glpi10/vendor/guzzlehttp/guzzle/src)
guzzlehttp/psr7 in (/home/test/public_html/glpi10/vendor/guzzlehttp/psr7/src)
glpi-project/inventory_format in (/home/test/public_html/glpi10/vendor/glpi-project/inventory_format/lib/php)
wapmorgan/unified-archive in (/home/test/public_html/glpi10/vendor/wapmorgan/unified-archive/src)
paragonie/sodium_compat in (/home/test/public_html/glpi10/vendor/paragonie/sodium_compat/src)
symfony/cache in (/home/test/public_html/glpi10/vendor/symfony/cache)
html2text/html2text in (/home/test/public_html/glpi10/vendor/html2text/html2text/src)
symfony/dom-crawler in (/home/test/public_html/glpi10/vendor/symfony/dom-crawler)
twig/twig in (/home/test/public_html/glpi10/vendor/twig/twig/src)
twig/string-extra in (/home/test/public_html/glpi10/vendor/twig/string-extra)
symfony/polyfill-ctype not found
symfony/polyfill-iconv not found
symfony/polyfill-mbstring not found
symfony/polyfill-php80 in (/home/test/public_html/glpi10/vendor/symfony/polyfill-php80)
phpCas version 1.3.8 in (/usr/share/php/CAS/source)

</details><details><summary>LDAP directories</summary><pre> 
Server: 'ldaps://test.test.com', Port: '636', BaseDN: 'OU=test,DC=test,DC=test,DC=com', Connection filter:
		'(&(objectClass=user)(objectCategory=person)(!(userAccountControl:1.2.840.113556.1.4.803:=2)))', RootDN:
		'CN=consult,OU=users,DC=test,DC=test,DC=com', Use TLS: none

</details><details><summary>SQL replicas</summary><pre> 
Not active

</details><details><summary>Notifications</summary><pre> 
Way of sending emails: SMTP+SSL (suport@test.com@smtp.gmail.com)

</details><details><summary>Plugins list</summary><pre> 
	formcreator          Name: Form Creator                   Version: 2.13.0-alpha.3 State: Installed / not activated
	mreporting           Name: More Reporting                 Version: 1.8.0-rc3  State: Installed / not activated
	telegrambot          Name: TelegramBot                    Version: 3.0.0      State: Installed / not activated

</pre></details>

**Comment by bolivarf** [comment]
An additional question in glpi 10 the login screen uses the language of the browser unlike previous versions of glpi which used the default language of glpi, this is intentional or it is a bug.

**Comment by cconard96** [comment]
I think it is intentional. I remember a pull request at some point adding the ability to use the browser's language when not logged in. I think it was done for the install process but may be used anytime you are not logged in.

I'll try to find the pull request and look into it.

Does it using the browser's language cause an issue?

**Comment by bolivarf** [comment]
I think knowing it beforehand isn't much of a problem, but it would be nice if there was a changelog entry indicating it.

**Comment by cconard96** [comment]
Fair enough.
It was added in 9.5.3.
Each release links to the milestone (the link usually says See changelog for details.) for that version which is more comprehensive than what usually ended up in the CHANGELOG file.

The 9.5.3 milestone:
https://github.com/glpi-project/glpi/milestone/44?closed=1

**Comment by trasher** [comment]
The pull request that introduced that change since GLPI 9.5.3:
https://github.com/glpi-project/glpi/pull/8254

This is part of the related version tag which is linked in release news (https://github.com/glpi-project/glpi/releases/tag/9.5.3) as "the changelog".

**Comment by trasher** [comment]
Curtis was a bit qiucker than me :)

**Comment by bolivarf** [comment]
@trasher @cconard96 Thank you very much for the information.

**Comment by cconard96** [comment]
Fixed via #11125
